.github/scripts/get_tutorials_stats.py

```diff
@@ -208,20 +221,34 @@ def main() -> None:
             "lines_added": int(i[5]),
             "lines_deleted": int(i[6]),
         }
-        for i in get_history_log
+        for i in tutorials_history_log + pytorch_docs_history_log
```

Comment:
Maybe we should add a repo parameter that makes it more clear where its coming from?  Idk if its being used anywhere so on one hand its probably safe to do but on the other hand there is no effect